RedHat-UX/red-hat-design-system
# fix(table): add container query support

## What I did

1. Switched from `@media` to `@container` in lightdom.css
2. Nested CSS
3. Switched to logical properties 

Closes #1476 

## Testing Instructions

1. View [deploy preview](https://deploy-preview-1881--red-hat-design-system.netlify.app/)
2. Please verify styles haven't been broken here given nesting of the CSS.  
3. Consider the implications of the @container query vs @media query.  Should we adjust the breakpoint?  Right now it is set at 768px.  That however maybe too large considering some content containers?  Should this drop to 576px instead now that we are taking into account only the width of the host element?

## Notes to Reviewers
